debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/LocalMapCommand.java

```diff
@@ -78,7 +78,7 @@ public void run(String command, String[] args, Context context, PrintStream out)
 			UDATA offsetPC = new UDATA(pc.sub(U8Pointer.cast(localMethod.bytecodes())));
 			CommandUtils.dbgPrint(out, "Relative PC = %d\n", offsetPC.longValue());
 
-			J9ClassPointer localClass = J9_CLASS_FROM_CP(localMethod.constantPool());
+			J9ClassPointer localClass = ConstantPoolHelpers.J9_CP_FROM_METHOD(localMethod).ramClass();
```

Comment:
You changed other places to use `ConstantPoolHelpers.J9_CLASS_FROM_METHOD()` but not here.

Reply:
I missed that one; I'll fix it and any others I can find.

Reply:
Done.